src/geo/lng_lat_bounds.js

```diff
@@ -83,7 +83,9 @@ class LngLatBounds {
 
         } else {
             if (Array.isArray(obj)) {
-                if (obj.every(Array.isArray)) {
+                // obj is LngLatBounds object if every element in the Obj array is also an array or
+                // if the LngLatBoundLike is defiend as a array of 4 numbers
```

Comment:
We usually add code comments to document non-obvious choices and higher-level problem solving. This comment doesn't seem necessary.